lesson04.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>Section 20: Seeing in nD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>